scipy/stats/_continuous_distns.py

```diff
@@ -5969,8 +5969,13 @@ def _ppf(self, q, a):
         return (a/(q**-a - 1.0))**(1.0/a)
 
     def _stats(self, a):
-        outputs = [None if i < a else np.nan for i in range(1, 5)]
+        outputs = [None if np.any(i < a) else np.nan for i in range(1, 5)]
         return outputs[:]
+    
+    def _mom1_sc(self, m, *args):
+        if np.any(np.array(args) >= m):
```

Comment:
```suggestion
        if np.any(m >= args[0]):
```
The inequality is backwards here. If the order of the moment (`i` in `_stats`, `m` here) is less than the shape parameter, we want to calculate the moment; otherwise it's `nan`.
Check me if I'm wrong, but `args` will always be a tuple with exactly one scalar element, the shape parameter of the distribution. If that is correct, then either this change or what you had originally will work. If that is incorrect, then  `np.any(np.array(...` is conceptually incorrect, think.